release-notes/v2.5.2.md

```diff
@@ -0,0 +1,43 @@
+# HPE CSI Driver for Kubernetes 2.5.2 Release Notes
+
+## HPE CSI Driver for Kubernetes 2.5.2
+
+| **Version:** |2.5.2|
+|--------------|-----|
+| **Revision:** | Monday January 13, 2025 |
+
+## Resolved Issues
+
+The following table lists the resolved issues for HPE CSI Driver for Kubernetes v2.5.2.
+
+| ID | Component | Title | Resolution |
+|----|-----------|-------|------------|
+| CON-3014 | csp.3par | Support LDAP users in Primera/3PAR. | Customers can use LDAP users for their Kubernetes admins. |
+| CON-3112 | csp.3par | Fixing excessive restarts of 3PAR CSP. | Fixed the issue by using a sync map instead of a normal map for avoiding concurrent issues. |
+| CON-3120 | csi.k8s | Remove volumeAccessMode from volumeAttributes and use volumeMode instead. | Support backup vendors that attaches block mode volume snapshots as filesystem to perform backup of KubeVirt. |
+| CON-3116 | csp.3par | Unable to restore from a snapshot when its parent PVC is deleted. | Primera CSP will not allow customer to delete a PVC when snapshots exists for that PVC. |
+| CON-3123 | csp.3par | PVC clone fails when multiple clone operation executed within 1 minute of interval. | Use unique names for the snapshots created as part of clone operations. |
+| CON-3126 | csp.3par | PVC creation failing for peer persistence enabled volumes with error "specified target is not a target of RCG". | Removed the extra validation which was looking for a match on target system name with the defined remote copy group target name. |
+| CON-3131 | csp.3par | CSI Driver stuck in a loop of dismissing VV from remote copy group when the sync operation happens between primary and secondary array. | Array now support admitting and dismissing volumes without stopping remote copy group and now CSP enhanced to adjust this. |
+| CON-3132 | csi.k8s | Remove the use of SSH and port 22 in CSI Driver. | Customer can use a modified secret with backend details of storage system IP along with the port if they want to avoid SSH. |
+| CON-3127 | csp.3par | Unpublish not removing VLUNs from secondary array for peer persistence enabled volumes. | Modifying the CRD with updated publish information in the case of adding a published volume to remote copy group through volume mutation. |
+| CON-3128 | csp.3par | Adding PVC to RCG fails when used with annotation method and Cloneof parameter. | Added mutation support for clone operation of volume in a remote copy group. |
+| CON-2549 | csi.k8s | Support volume expansion for RWX PVCs. | Handle the RWX resize request and make sure the corresponding RWO volume get expanded. |
```

Comment:
It is not just RWX, this enhancement supports expansion of all NFS volumes (all access modes)

Reply:
Addressed the comment